tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -90,7 +90,7 @@ def load_dut_basic_facts(session):
 
         dut_name = tbinfo['duts'][0]
         inv_name = tbinfo['inv_name']
-        ansible_cmd = 'ansible -m dut_basic_facts -i {} {} -o'.format(inv_name, dut_name)
+        ansible_cmd = 'ansible -m dut_basic_facts -i ../ansible/{} {} -o'.format(inv_name, dut_name)
```

Comment:
It may be better if path is found by first tracing the current path, and then going to ansible dir. 
```suggestion
        file_path = `os.path.dirname(os.path.realpath(__file__))`
        ansible_cmd = 'ansible -m dut_basic_facts -i {}/../../../../ansible/{} {} -o'.format(file_path, inv_name, dut_name)
```

Reply:
True. But we always run test from tests/, so ../ is practically the same.